examples/byoi/ubuntu-fips/Dockerfile

```diff
@@ -0,0 +1,148 @@
+
+# Kairos framework packages for ubuntu fips
+FROM quay.io/kairos/framework:master_ubuntu-20-lts-fips as kairos-fips
+
+# Base ubuntu image (focal)
+FROM ubuntu:focal as base
+
+# Generate os-release file
+FROM quay.io/kairos/osbuilder-tools:latest as osbuilder
+RUN zypper install -y gettext
+RUN mkdir /workspace
+COPY --from=base /etc/os-release /workspace/os-release
+# You should change the following values according to your own versioning and other details
+RUN OS_NAME=kairos-core-ubuntu-fips \
+  OS_VERSION=v9.9.9 \
+  OS_ID="kairos" \
+  OS_NAME=kairos-ubuntu-fips \
+  BUG_REPORT_URL="https://github.com/YOUR_ORG/YOUR_PROJECT/issues" \
+  HOME_URL="https://github.com/YOUR_ORG/YOUR_PROJECT" \
+  OS_REPO="quay.io/YOUR_ORG/ubuntu-fips" \
+  OS_LABEL="latest" \
+  GITHUB_REPO="YOUR_ORG/YOUR_PROJECT" \
+  VARIANT="fips" \
+  FLAVOR="ubuntu" \
+  /update-os-release.sh
+
+# Build the custom ubuntu image
+FROM base
+
+# Don't get asked while running apt commands
+ENV DEBIAN_FRONTEND=noninteractive
+
+## THIS comes from the Ubuntu documentation: https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/tutorials/create_a_fips_docker_image.html
+## I've just added "linux-image-fips" in the package list
+RUN --mount=type=secret,id=pro-attach-config \
+    apt-get update \
+    && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
+    && pro attach --attach-config /run/secrets/pro-attach-config \
+    && apt-get upgrade -y \
+    && apt-get install -y openssl libssl1.1 libssl1.1-hmac libgcrypt20 libgcrypt20-hmac strongswan strongswan-hmac openssh-client openssh-server linux-image-fips
+
+
+## Kairos setup
+## From documentation: https://kairos.io/docs/reference/build-from-scratch/
+RUN mkdir -p /run/lock
+RUN mkdir -p /usr/libexec
+RUN touch /usr/libexec/.keep
+
+## Kairos required packages
+## See: https://github.com/kairos-io/kairos/blob/master/images/Dockerfile.ubuntu-20-lts
+RUN apt-get install -y --no-install-recommends \
```

Comment:
shouldnt this be before the fips stuff?

Reply:
I don't think it really matters, I moved it down _just_in_case_ in the repos there were some of these packages, and so to give a chance to get replaced with fips-ones.